package.json

```diff
@@ -124,6 +126,7 @@
     "bundle": "StandaloneHTML ./dist/index.html ./dist/ItkVtkViewer.html",
     "commit": "git cz",
     "format": "prettier --write src/UserInterface/**/*.js src/*.js",
+    "lint:types": "tsc --noEmit",
```

Comment:
Can this be part of the `test` npm script?

Reply:
Done.